athena/map/PlainMap.tsx

```diff
@@ -28,6 +28,7 @@ export type PlainMapConfig = Readonly<{
   ];
   seedCapital: number;
   winConditions?: LegacyWinConditions;
+  initialCharge: number;
```

Comment:
Since this is a new field not present in existing maps, it needs to be typed as `number | undefined`.

Reply:
Done. Updates made to PlainMap.tsx and startGame.tsx.